x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.test.ts

```diff
@@ -514,7 +592,7 @@ describe('build_exceptions_query', () => {
         entries,
       });
       const expectedQuery =
-        'b:("value-1" OR "value-2") AND parent:{ nestedField:"value-3" } AND NOT _exists_e';
+        'b:("value-1" OR "value-2") AND parent:{ NOT nestedField:"value-3" } AND NOT _exists_e';
```

Comment:
👍

Reply:
I think I missed where this logic changed, but the expected query looks correct. 🚀

Reply:
Changed in `x-pack/plugins/security_solution/common/detection_engine/build_exceptions_query.ts` line 129